examples/va/nfv/hci-ovs-dpdk-sriov/dataplane-post-ceph.md

```diff
@@ -0,0 +1,91 @@
+# Configuring and deploying the post-Ceph dataplane
+
+## Assumptions
+
+- The pre-Ceph [dataplane](dataplane-pre-ceph.md) was already deployed and Ceph was manually installed afterwords
+
+## Initialize post-Ceph
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the hci directory
+```
+cd architecture/examples/va/nfv/hci-ovs-dpdk-sriov
+```
+Edit the [values.yaml](values.yaml) and [service-values.yaml](service-values.yaml)
+files to suit your environment.
+```
+vi values.yaml
+vi service-values.yaml
+```
+The ceph sections of [values.yaml](values.yaml) should have values like this.
+```yaml
+data:
+    ceph:
+        conf: $CONF
+        keyring: $KEY
+    nova:
+      ceph:
+        conf:
+            conf: |
+              [libvirt]
+              images_type=rbd
+              images_rbd_pool=vms
+              images_rbd_ceph_conf=/etc/ceph/ceph.conf
+              images_rbd_glance_store_name=default_backend
+              images_rbd_glance_copy_poll_interval=15
+              images_rbd_glance_copy_timeout=600
+              rbd_user=openstack
+              rbd_secret_uuid=$FSID
```

Comment:
I know this was probably copy-pasted from the HCI VA, but the indentation looks off here.

Reply:
fixed